src/fs.rs

```diff
@@ -20,15 +20,25 @@
 //! Directories only need a corresponding directory in the host filesystem if
```

Comment:
Oh, the paragraph above this is outdated now that we have ZIP files. Uhh, I think I'll try to rewrite it myself later if that's okay.

Reply:
Sure